Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM hashicorp/terraform:light
+RUN apk update && apk add ansible bash
+ADD https://github.com/radekg/terraform-provisioner-ansible/releases/download/v2.0.1/terraform-provisioner-ansible-linux-amd64_v2.0.1 /root/.terraform.d/plugins/terraform-provisioner-ansible
```

Comment:
@adamwg do you think it would be possible to use an environment variable for: version, OS family and architecture? It would be possible to override the version when building using `--build-arg`, like this:

    docker build --build-arg TAP_VERSION=2.0.1 --build-arg TAP_OS=darwin TAP_ARCH=amd64 -t terraform-ansible  .

The defaults being `2.0.1`, `linux` and `amd64` would be fine, I guess.

Reply:
Implemented in https://github.com/radekg/terraform-provisioner-ansible/commit/1344c390008e793526e4612c4cad357ccc6419e6